clang/test/CIR/CodeGen/builtin-arm-ldrex.c

```diff
@@ -0,0 +1,21 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-android24  -fclangir -emit-cir -mmlir --mlir-print-ir-before=cir-lowering-prepare %s -o %t.cir 2>&1 | FileCheck %s
```

Comment:
You don't need `-mmlir --mlir-print-ir-before=cir-lowering-prepare` here, nor redirecting the stderr to stdout (this is only necessary when checking the output of intermediate passes). I suggest you follow similar to `clang/test/CIR/CodeGen/aarch64-neon-intrinsics.c`

```
// RUN: %clang_cc1 ... -triple arm64-none-linux-gnu -target-feature +neon ... -o %t.cir
// RUN: FileCheck --check-prefix=CIR --input-file=%t.cir %s
```
In the future we will add a `LLVM` check, so let's save the work.